docs/server/source/quickstart.md

```diff
@@ -6,34 +6,56 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 [Install MongoDB Server 3.4+](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
 
-B. Run MongoDB. Open a Terminal and run the command:
+Default database directory where MongoDB stores its data: /data/db.
+
+B. Run MongoDB with default database path i.e. /data/db, open a Terminal and run the following commands as root/admin user:
+```text
+$ sudo su
+$ mkdir -p /data/db
```

Comment:
We use lines like `$ sudo command` in the rest of this page so let's do the same here, i.e. remove the `$ sudo su` and just have `$ sudo mkdir -p /data/db`.